### PR TITLE
Developer console

### PR DESCRIPTION
**Pull request by AlexanderK22**
Translated the "Developer console" section

### CODE REVIEW

File: 1-js/01-getting-started/4-devtools/article.md [hunk 1]
```diff
@@ -1,63 +1,64 @@
-# Developer console
+# Entwickler-Konsole
 
-Code is prone to errors. You will quite likely make errors... Oh, what am I talking about? You are *absolutely* going to make errors, at least if you're a human, not a [robot](https://en.wikipedia.org/wiki/Bender_(Futurama)).
+Code ist anfällig für Fehler. Du wirst sehr wahrscheinlich Fehler machen... Oh, was rede ich da? Du wirst *auf jeden Fall* Fehler machen, zumindest wenn du ein Mensch bist und kein [Roboter](https://en.wikipedia.org/wiki/Bender_(Futurama)).
 
-But in the browser, users don't see errors by default. So, if something goes wrong in the script, we won't see what's broken and can't fix it.
+Aber in Browsern sehen Benutzer standardmäßig keine Fehler. Also, wenn etwas im Skipt schief läuft, sehen wir nicht was falsch ist und können es nicht beheben.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Aber in Browsern sehen Benutzer standardmäßig keine Fehler. Also, wenn etwas im Skript schief läuft, sehen wir nicht was falsch ist und können es nicht beheben.
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 2]
```diff
@@ -1,63 +1,64 @@
...
 
-To see errors and get a lot of other useful information about scripts, "developer tools" have been embedded in browsers.
+Um Feheler zu sehen und weitere nützliche Informationen über Skripts zu bekommen, wurden "Entwicklertools" in Browsern eingebaut.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Um Fehler zu sehen und weitere nützliche Informationen über Skripts zu bekommen, wurden "Entwicklertools" in Browser eingebaut.
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 3]
```diff
@@ -1,63 +1,64 @@
...
 
-Most developers lean towards Chrome or Firefox for development because those browsers have the best developer tools. Other browsers also provide developer tools, sometimes with special features, but are usually playing "catch-up" to Chrome or Firefox. So most developers have a "favorite" browser and switch to others if a problem is browser-specific.
+Die meisten Entwickler tendieren zu Chrome oder Firefox für die Entwicklung, weil diese Browser die besten Entwicklertools haben. Andere Browser stellen ebenfalls Entwicklertools bereit, manchmal mit besonderen Eigenschaften aber in der Regel lassen Chrome oder Firefox diese hinter sich. So haben die meisten Entwickler eine "Lieblingsbrowser" und wechseln zu einen anderen, wenn es ein browserspezifisches Problem gibt.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Die meisten Entwickler tendieren zu Chrome oder Firefox für die Entwicklung, weil diese Browser die besten Entwicklertools haben. Andere Browser stellen ebenfalls Entwicklertools bereit, manchmal mit besonderen Eigenschaften aber in der Regel lassen Chrome oder Firefox diese hinter sich. So haben die meisten Entwickler einen "Lieblingsbrowser" und wechseln zu einen anderen, wenn es ein browserspezifisches Problem gibt.
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 4]
```diff
@@ -1,63 +1,64 @@
...
 
-Developer tools are potent; they have many features. To start, we'll learn how to open them, look at errors, and run JavaScript commands.
+Entwicklertools sind mächtig; sie haben viele Features. Zuerst lernen wir, wie man sie öffnet, sich Fehler ansieht und JavaScript Befehle ausführt
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Entwicklertools sind leistungsstark; sie haben viele Features. Zuerst lernen wir, wie man sie öffnet, sich Fehler ansieht und JavaScript-Befehle ausführt
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 5]
```diff
@@ -1,63 +1,64 @@
...
 
 ## Google Chrome
 
-Open the page [bug.html](bug.html).
+Öffne die Seite [bug.html](bug.html).
 
-There's an error in the JavaScript code on it. It's hidden from a regular visitor's eyes, so let's open developer tools to see it.
+Dort ist ein Fehler im JavaScript Code. Er ist aus Sich eines normalen Besuchers nicht zu sehen, also öffnen wird die Entwicklertools um ihn zu sehen.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Dort ist ein Fehler im JavaScript-Code. Er ist aus Sicht eines normalen Besuchers nicht zu sehen, also öffnen wird die Entwicklertools um ihn zu sehen.
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 6]
```diff
@@ -1,63 +1,64 @@
...
 
-Press `key:F12` or, if you're on Mac, then `key:Cmd+Opt+J`.
+Drücke `key:F12` oder wenn du auf einem Mac bist `key:Cmd+Opt+J`.
 
-The developer tools will open on the Console tab by default.
+Die Entwicklertools öffnen sich standardmäßig auf dem Konsolen-Reiter.
 
-It looks somewhat like this:
+Es sieht ungefähr wie das aus:
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Es sieht ungefähr so aus:
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 7]
```diff
@@ -1,63 +1,64 @@
...
 
 ![chrome](chrome.png)
 
-The exact look of developer tools depends on your version of Chrome. It changes from time to time but should be similar.
+Wie genau die Entwicklertools aussehen, hängt von der Chrome Version ab. Sie ändern sich von Zeit zu Zeit aber sollten ähnlich aussehen.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Wie genau die Entwicklertools aussehen, hängt von der Chrome-Version ab. Sie ändern sich von Zeit zu Zeit, aber sollten ähnlich aussehen.
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 8]
```diff
@@ -1,63 +1,64 @@
...
 
-- Here we can see the red-colored error message. In this case, the script contains an unknown "lalala" command.
-- On the right, there is a clickable link to the source `bug.html:12` with the line number where the error has occurred.
+- Hier sehen wir die rot markierte Fehlermeldung. In diesem Fall beinhaltet das Skript einen unbekannten Befehl "lalala".
+- Auf der rechten Seite ist ein anklickbarer Link zur Quelle `bug.html:12` mit der Zeilennummer, wo der Fehler aufgetreten ist.
 
-Below the error message, there is a blue `>` symbol. It marks a "command line" where we can type JavaScript commands. Press `key:Enter` to run them.
+Unter der Fehlermeldung befindet sich ein Blaues `>` Symbol. Es markiert eine "Befehlszeile" wo wir JavaScript Befehle eingeben können. Drücke `key:Enter` um sie auszuführen.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Unter der Fehlermeldung befindet sich ein blaues `>` Symbol. Es markiert eine "Befehlszeile" wo wir JavaScript-Befehle eingeben können. Drücke `key:Enter` um sie auszuführen.
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 9]
```diff
@@ -1,63 +1,64 @@
...
+
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]: Leave a line blank, please. It should match the original formatting.

File: 1-js/01-getting-started/4-devtools/article.md [hunk 10]
```diff
@@ -1,63 +1,64 @@
...
+Nun können wir Fehler sehen und das ist genug für den Anfang. Wir kehren späte zu den Entwicklertoos zurück und behandeln das Debugging im Kapitel <info:debugging-chrome> tiefer.
 
-Now we can see errors, and that's enough for a start. We'll come back to developer tools later and cover debugging more in-depth in the chapter <info:debugging-chrome>.
 
 ```smart header="Multi-line input"
-Usually, when we put a line of code into the console, and then press `key:Enter`, it executes.
+Normalerweise, wenn wir eine Zeile Code in die Konsole eingeben und dann `key:Enter` drücken, wird er ausgeführt.
 
-To insert multiple lines, press `key:Shift+Enter`. This way one can enter long fragments of JavaScript code.
+Um mehrere Zeilen einzugeben, drücke `key:Shift+Enter`. Auf diese Weise können längere Fragmente JavaScript Code eingegeben werden.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Um mehrere Zeilen einzugeben, drücke `key:Shift+Enter`. Auf diese Weise können längere Fragmente JavaScript-Code eingegeben werden.
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 11]
```diff
@@ -1,63 +1,64 @@
...
-Most other browsers use `key:F12` to open developer tools.
+Die meisten Browser benutzen `key:F12` um die Entwicklertools zu öffnen.
 
-The look & feel of them is quite similar. Once you know how to use one of these tools (you can start with Chrome), you can easily switch to another.
+Das Aussehen ist meist sehr ähnlich. Sobald du weißt wie man eins dieser Tools benutzt (du kannst mit Chrome anfangen), kannst du einfach zu einem anderen wechseln.
 
 ## Safari
 
-Safari (Mac browser, not supported by Windows/Linux) is a little bit special here. We need to enable the "Develop menu" first.
+Safari (Mac Browser, nicht unterstützt werden Windows/Linux) ist etwas speziell. Wir müssen zuerst das "Entwickler Menü" aktivieren.
```
Comment (christianhegedues) [inline 1-js/01-getting-started/4-devtools/article.md]:
```suggestion
Safari (Mac Browser, nicht unterstützt werden Windows/Linux) ist etwas speziell. Wir müssen zuerst das "Entwickler-Menü" aktivieren.
```

File: 1-js/01-getting-started/4-devtools/article.md [hunk 12]
```diff
@@ -1,63 +1,63 @@
-# Developer console
+# Entwickler-Konsole
 
-Code is prone to errors. You will quite likely make errors... Oh, what am I talking about? You are *absolutely* going to make errors, at least if you're a human, not a [robot](https://en.wikipedia.org/wiki/Bender_(Futurama)).
+Code ist anfällig für Fehler. Du wirst sehr wahrscheinlich Fehler machen... Oh, was rede ich da? Du wirst *auf jeden Fall* Fehler machen, zumindest wenn du ein Mensch bist und kein [Roboter](https://en.wikipedia.org/wiki/Bender_(Futurama)).
```
Comment (mka4975) [inline 1-js/01-getting-started/4-devtools/article.md]: Maybe change the wiki page to the german page

Comment (AlexanderK22) [reply 1-js/01-getting-started/4-devtools/article.md]: There is currently no corresponding German wiki page for it.

**Pull request merged**